tests/test_models/test_voxel_encoders/test_voxel_encoders.py

```diff
@@ -15,3 +16,27 @@ def test_hard_simple_VFE():
 
     outputs = hard_simple_VFE(features, num_voxels, None)
     assert outputs.shape == torch.Size([240000, 5])
+
+
+def test_seg_VFE():
+    if not torch.cuda.is_available():
+        pytest.skip('test requires GPU and torch+cuda')
+    seg_VFE_cfg = dict(
+        type='SegVFE',
+        feat_channels=[64, 128, 256, 256],
+        with_voxel_center=True,
+        feat_compression=16,
+        return_point_feats=True)
+    seg_VFE = MODELS.build(seg_VFE_cfg)
+    seg_VFE = seg_VFE.cuda()
+    features = torch.rand([240000, 6]).cuda()
+    coors = []
+    for i in range(4):
+        coor = torch.randint(0, 10, (60000, 3))
+        coor = F.pad(coor, (1, 0), mode='constant', value=i)
+        coors.append(coor)
+    coors = torch.cat(coors, dim=0).cuda()
+    out_features, out_coors, out_point_features = seg_VFE(features, coors)
+    assert out_features.shape[0] == out_coors.shape[0]
+    assert len(out_point_features) == 5
```

Comment:
After solve above suggestion:

```suggestion
    assert len(out_point_features) == 4
```